src/shared/components/SubmissionManagement/SubmissionManagement/index.jsx

```diff
@@ -78,9 +78,13 @@ export default function SubmissionManagement(props) {
           </a>
         </div>
         <div styleName="right-col">
-          <p styleName="round">
-            {currentPhase.name}
-          </p>
+          {
+            currentPhase && (
+            <p styleName="round">
+              {currentPhase.name}
+            </p>
+            )
+          }
           {
             challenge.status !== 'COMPLETED' ? (
```

Comment:
Check this validation, this was changed from `COMPLETED` to `Completed`

Reply:
> The challenge has ended - instead that showing 0H 0M left

Looks like its because of this line, changing this to `Completed` should solve it

> Ends: Thursday 09/03/20 10:06 PM

After my changes, if challenge is ended, you should not see this text. And if it's visible (means challenge still open), it should show phase name as well: `<PhaseName> Ends Thursday 09/03/20 10:06 PM` So, you'll either see nothing at there, or will see the phase name as well. Where did you get this?

Reply:
@cagdas001 Yep, looks right

Reply:
@luizrrodrigues pushed the new commit for `COMPLETED` -> `Completed` change, PR will be available after automated tests